site/docs/zh/hosting/deno-deploy.md

```diff
@@ -0,0 +1,59 @@
+# 托管：Deno deploy
```

Comment:
Maybe we should use `Deno Deploy` in the title?
@KnorpelSenf

Reply:
We definitely should, you're right!